py_ecc/bls/typing.py

```diff
@@ -0,0 +1,17 @@
+from typing import (
+    NewType,
+    Tuple,
+)
+
+from py_ecc.fields import (
+    optimized_bls12_381_FQ as FQ,
+    optimized_bls12_381_FQ2 as FQ2,
+)
+
+
+# TODO: FQ and FQ2 here are invalid types, they need to be fixed in the future
+G1Uncompressed = NewType('G1Uncompressed', Tuple[FQ, FQ, FQ])
```

Comment:
I think you can use `Optimized_FQPoint3D` instead of `Tuple[FQ, FQ, FQ]` here. You can try adding this, and see what `mypy` thinks about this. If there is an error, I'll take care in the future, when the refactor is over and the goal is to 
re-enable `type checking`.

Reply:
Thanks @Bhargavasomu. I've tried `Optimized_FQPoint3D`, but mypy still unhappy about the     FQ and FQ2 used in `utils.py`. When I change them to those from optimized_bn128, the curve is then unhappy about it.
Will wait for you refactor in the future and give up doing typing in this PR for now.